master/reference/typha/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring Typha
+canonical_url: 'https://docs.projectcalico.org/v3.1/reference/typha/configuration'
```

Comment:
This is perfect, we have the right canonical_url that htmlproofer will complain as a chicken-egg problem and the error resolves upon merge.